substrate/frame/sudo/src/weights.rs

```diff
@@ -55,6 +55,7 @@ pub trait WeightInfo {
 	fn sudo() -> Weight;
 	fn sudo_as() -> Weight;
 	fn remove_key() -> Weight;
+	fn check_only_sudo_account() -> Weight;
```

Comment:
my question still persists: will a bot be able to correctly figure this out? or will this have to be added manually?

Reply:
Everything that is benchmarked in this PR is auto-generated by the bench script.